README.md

```diff
@@ -10,27 +10,38 @@ This module requires the following modules/libraries:
 
 * [Islandora](https://github.com/islandora/islandora)
 
-This module has the following as optional requirements for certain features:
+This module has also the following optional requirements for certain features:
+
+For Solr Search Integration, which allows the actual use of the 'Solr filter query' option and enables an aditional Solr Driven 'Compound Member Query' option:
+
+* [Islandora Solr Search](https://github.com/Islandora/islandora_solr_search) Module
 
 For Islandora Compound Object JAIL Display:
 
-* [JAIL](https://github.com/sebarmeli/JAIL) library 
+* [JAIL](https://github.com/sebarmeli/JAIL) JQuery library 
 
 ## Installation
 
 Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
 
-If utilizing the lazy loading image ability of the solution pack, the [JAIL](https://github.com/sebarmeli/JAIL)
+If utilizing the lazy loading image ability of the solution pack, the [JAIL](https://github.com/sebarmeli/JAIL) JQuery
 library must be present within sites/all/libraries/JAIL.
 
 ## Configuration
 
-Set the 'Child relationship predicate' and 'Solr filter query', as well as select options in Administration » Islandora » Solution pack configuration » Compound Object Solution Pack (admin/islandora/solution_pack_config/compound_object).
+Set the 'Child relationship predicate' and 'Solr filter query', as well as display options in search results and derivative creation options in Administration » Islandora » Solution pack configuration » Compound Object Solution Pack (admin/islandora/solution_pack_config/compound_object).
+
+The 'Redirect to first child when a compound object is requested' option skips the Islandora generated parent object display by redirecting, via HTTP, to the first visible child of a compound ensemble.
 
 Optionally, enable the JAIL compound block to utilize the lazy loading image
-ability as outlined below.
+ability using Javascript by checking 'Use alternative, alternative dsiplay for compounds' as shown below.
+
+
+![Configuration](docs/compound-basic-settings.png)
+
+### Solr Integration
 
-![Configuration](https://user-images.githubusercontent.com/25011926/39889778-d1a91aca-5466-11e8-8eb1-1978cac81104.png)
+If the Solr Search Module is enabled, and additional Compound Member Query Option and a new 'Solr Backend' Menu tab will be activated at Administration » Islandora » Solution Pack Configuration » Compound Object Solution Pack (admin/islandora/solution_pack_config/compound_object/solr)
```

Comment:
Thanks Diego! Did you mean "an additional" instead of "and additional"?

Reply:
Are you mentalist 😄 ? Yes!! Fixing after lunch, thanks so much for proofreading!!